src/model/Subcategory.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+import utils.Validation;
+
+public class Subcategory {
+
+    private String title;
+    private String slug;
+    private String description;
+    private String studyGuide;
+    private boolean enabled;
+    private Integer order;
+    private String iconPath;
+    private String colorHexCode;
+    private model.Category parentCategory;
```

Comment:
Estranho usar o nome completo da classe (pacote + nome da classe) aqui. Elas estão no mesmo pacote!

```suggestion
    private Category parentCategory;
```

Reply:
Provavelmente foi o IntelliJ durante as alterações que fiz no design de Categoria. Vou consertar :)